### ISSUE TITLE
Using Custom Dataset with Different Input Lengths for Model

### ISSUE THREAD

**Issue by rozhix** [thuml/Nonstationary_Transformers#41]
My input dimensions are [1500, 50, 6] and my label dimensions are [1500, 1]. Can I use this model? If so, could you please explain how to configure it correctly?

**Comment by WenWeiTHU** [comment]
It looks like a classification task as you mentioned the "label"

**Comment by WenWeiTHU** [comment]
If it means that the task only cares about the predicted value of one of the variables (with the other inputs as covariates), you can try to calculate the prediction error of only the target variable and backpropagate it when the loss function is optimized.
